### ISSUE TITLE
Login command is broken

### ISSUE THREAD

**Issue by davidcollom** [maelvls/clockidup#2]
I think the login process flow may be slightly broken:

```~$ clockidup -debug  login
info: the API token is available at https://clockify.me/user/settings
debug: curl -v -X GET  --header 'X-Api-Key: ' --header 'host: api.clockify.me'   https://api.clockify.me/api/v1/workspaces
? Clockify API token [REDACTED]
debug: token: [REDACTED]
debug: curl -v -X GET  --header 'X-Api-Key: [REDACTED]' --header 'host: api.clockify.me'   https://api.clockify.me/api/v1/workspaces
debug: curl -v -X GET  --header 'X-Api-Key: ' --header 'host: api.clockify.me'   https://api.clockify.me/api/v1/workspaces
error: login failed: token seems to be invalid
```

It looks as though, the configfile is either not written or not re-read in on the second api call?

If I create the file manually: `${HOME}/.config/standup.yaml` all is well and things start working as expected.

**Comment by maelvls** [comment]
Looks like I introduced a "hidden shared state" bug. I used the same `http.DefaultClient` in two calls: the first call would "wrap" the transport in `http.DefaultClient` a first time, and the second call would wrap a second time; the most nested one (i.e., the first call to `NewClockify`) would override the token of the least nested transport...

Ahh if only the use of `http.DefaultClient` was deprecated... Also, http.DefaultTransport should have some creation function to avoid state sharing...

I fixed the bug in [v0.3.0](https://github.com/maelvls/clockidup/releases/tag/v0.3.0), and you can install it with:

```sh
go install github.com/maelvls/clockidup@latest
```

To check that you have the latest version:

```sh
❯ go version -m $(which clockidup)
/home/mvalais/go/bin/clockidup: go1.16.3
        path    github.com/maelvls/clockidup
        mod     github.com/maelvls/clockidup    v0.3.0
```

**NOTE:** I changed the name of the config file from `~/.config/standup.yml` to `~/.config/clockidup.yml`.

Let me know if that works better or not!

**Comment by davidcollom** [comment]
:+1: Confirmed working by re-installing and logging in again.

Thanks for the speedy response! approve the config file name change too makes total sense! 👍 

Thanks!
